<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>450</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Solve</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_6">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_7">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_8">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>40</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_9">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_11">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_12">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_13">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_14">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_15">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_16">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_17">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>80</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_18">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>80</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_19">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_20">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_21">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_22">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_23">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_24">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_25">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>120</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_26">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_27">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>120</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_28">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_29">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_30">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_31">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_32">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_33">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_34">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>170</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_35">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>170</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_36">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>170</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_37">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_38">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_39">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_40">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_41">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_42">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_43">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_44">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>210</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_45">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>210</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_46">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_47">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_48">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_49">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_50">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_51">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_52">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>250</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_53">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>250</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_54">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>250</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_55">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_56">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_57">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>300</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_58">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_59">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>300</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_60">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>300</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_61">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>300</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_62">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>300</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_63">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>300</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_64">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_65">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_66">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>340</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_67">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>340</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_68">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>340</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_69">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>340</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_70">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>340</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_71">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>340</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_72">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>340</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_73">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_74">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>380</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_75">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>380</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_76">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>380</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_77">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>380</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_78">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>380</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_79">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>380</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_80">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>380</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_81">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>380</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
